test/vinyl/gh-5823-crash_snapshot.lua

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env tarantool
+
+--
+-- mode == 0: casual bootstrap;
+-- mode == 1: force recovery bootstrap;
+-- mode == 2: casual bootstrap and fill in data.
+--
+local mode = tonumber(arg[1])
+box.cfg ({
+    replication = os.getenv("MASTER"),
+    vinyl_memory = 128 * 1024 * 1024,
```

Comment:
Do you really need a specific `vinyl_memory` setting?

Reply:
In fact - no. Removed